MapboxNavigation/RouteMapViewController.swift

```diff
@@ -287,6 +291,32 @@ class RouteMapViewController: UIViewController {
             mapView.removeArrow()
         }
     }
+
+    func updateCameraAltitude(for routeProgress: RouteProgress) {
+        guard mapView.tracksUserCourse else { return } //only adjust when we are actively tracking user course
+        
+        let zoomOutAltitude = RouteMapViewController.zoomedOutManeuverAltitude
+        let defaultAltitude = mapView.defaultAltitude
```

Comment:
Would it make sense for `mapView.defaultAltitude ` and `RouteMapViewController.zoomedOutManeuverAltitude ` to live next to one another?